ubuntu-personal-store/ubuntu-personal-store.qml.in

```diff
@@ -49,6 +49,11 @@ ApplicationWindow {
     id: webView
     focus: true
 
+    context: WebContext {
+        dataPath: Qt.application.arguments[1]
```

Comment:
I find it a bit dangerous to hardcode this, ... we might decide at some point to change the command line args and this might fail silently (since it is not really part of the typical qmlscene args)?